modules/ROOT/pages/batch-processing-concept.adoc

```diff
@@ -112,16 +112,19 @@ This first phase is implicit. During this phase, the runtime performs all the "b
 Mule exposes the batch job instance ID through the `batchJobInstanceId` variable. This variable is available in every step and the on-complete phase.
 +
 . Mule creates a persistent queue and associates it with the new batch job instance.
++
+Queue names are prefixed with `BSQ`, and you can see them in Anypoint Runtime Manager:
```

Comment:
this is confusing. Above it says that ONE queue is created (which is correct), but here says `queue names` in plural. It becomes unclear how many queues per job instance are actually created. 

Also, one job instance can also create additional queues during the process phase. For example, two additional queues are created per each streaming aggregator. Shouldn't that be mentioned too?